Tests/BSONTests/ExtendedJSONConversionTests.swift

```diff
@@ -370,10 +363,10 @@ open class ExtendedJSONConversionTestCase: BSONTestCase {
             .to(equal(date))
         expect(date.toCanonicalExtendedJSON()).to(equal(["$date": ["$numberLong": "500004"]]))
         // Relaxed Success case
-        let date2 = Date(msSinceEpoch: 1_356_351_330_501)
```

Comment:
could we add a new case instead of removing the old one here?

Reply:
sure